src/yunohost/app.py

```diff
@@ -2771,16 +2771,16 @@ def _assert_system_is_sane_for_app(manifest, when):
 
     # Some apps use php-fpm or php5-fpm which is now php7.0-fpm
```

Comment:
` # Some apps use php-fpm or php5-fpm which is now php7.0-fpm`

Souldn't this be `"[...] which is now php7.3-fpm"` 😉